app/views/catalog/facet.html.erb

```diff
@@ -4,8 +4,12 @@
 
 <div data-controller="facet-filter">
   <%= render BlacklightModalComponent.new do |component| %>
-    <% component.header { facet_field_label(@facet.key) } %>
+    <% component.header do %>
+      <%= facet_field_label(@facet.key) %>
+      <a class="copy-button" data-controller="copy" data-action="copy#copy" aria-label="Copy this list" href="#"><i class="bi bi-clipboard" title="Copy this list"></i></a>
```

Comment:
It looks like other Bootstrap icons use `<span>` instead of `<i>` in Argo currently. Worth standardizing on that here or does using `<i>` have affordances here?

Reply:
Thanks for finding that.  I don't any benefit to `<i>` ... was just what I copied from website example.  Will change to `<span>`